site/content/integrate/incoming-webhooks/_index.md

```diff
@@ -68,6 +68,7 @@ Incoming webhooks support more than just the `text` field. Here is a full list o
 | attachments | [Message attachments](https://docs.mattermost.com/developer/message-attachments.html) used for richer formatting options. | If `text` is not set, yes |
 | type | Sets the post `type`, mainly for use by plugins.<br> If not blank, must begin with "custom\_". | No |
 | props | Sets the post `props`, a JSON property bag for storing extra or meta data on the post.<br> Mainly used by other integrations accessing posts through the REST API.<br> The following keys are reserved: "from\_webhook", "override\_username", "override\_icon\_url", "override\_icon\_emoji", "webhook\_display\_name" and "attachments". | No |
```

Comment:
I miss the `card` field as reserved word here.

Reply:
I added the 'card' field to reserved words.